aries_vcx/src/messages/thread.rs

```diff
@@ -1,9 +1,8 @@
 use std::collections::HashMap;
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Clone)]
+#[derive(Debug, Deserialize, Serialize, PartialEq, Clone, Default)]
 pub struct Thread {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub thid: Option<String>,
+    pub thid: String,
```

Comment:
I suppose you already have reverted this locally

Reply:
Yes.